runtime/runtime.go

```diff
@@ -540,7 +542,14 @@ func (r *interpreterRuntime) parseAndCheckProgram(
 				sema.WithPredeclaredValues(valueDeclarations),
 				sema.WithPredeclaredTypes(typeDeclarations),
 				sema.WithValidTopLevelDeclarationsHandler(validTopLevelDeclarations),
-				sema.WithLocationHandler(runtimeInterface.ResolveLocation),
+				sema.WithLocationHandler(func(identifiers []Identifier, location Location) (res []ResolvedLocation) {
+					var err error
+					res, err = runtimeInterface.ResolveLocation(identifiers, location)
```

Comment:
Just realizing that the call to the runtime interface should be wrapped, just like in all other cases:

```suggestion
					wrapPanic(func() {
						res, err = runtimeInterface.ResolveLocation(identifiers, location)
					})
```